server/plugin.go

```diff
@@ -394,3 +397,112 @@ func (p *Plugin) dm(userID string, message string) error {
 	}
 	return nil
 }
+
+func (p *Plugin) setUserStatus(userID string, meetingID int, meetingEnd bool) error {
+	statusChangePrefkey := fmt.Sprintf("%v_%v", changeStatusKey, userID)
+	changeStatus, err := p.API.KVGet(statusChangePrefkey)
+	if err != nil {
+		p.API.LogDebug("Could not get old status from KVStore", "err", err.Error())
+		return err
+	}
+
+	if string(changeStatus) != yes {
+		return nil
+	}
+
+	statusKey := fmt.Sprintf("%v%v", oldStatusKey, meetingID)
+	if meetingEnd {
+		b, appErr := p.API.KVGet(statusKey)
```

Comment:
Is there a better variable name than `b`?

Reply:
@cvhariharan What do you think about this?

Reply:
Oh, sorry. Missed it.

Reply:
This was resolved